src/index.ts

```diff
@@ -274,6 +293,21 @@ const parseArguments = () => {
         return result;
     };
 
+    const parseKilo = (value: string): number => {
```

Comment:
please remove this and make the argument units kilobytes: "the size, in kilobytes of the blah".